README.md

```diff
@@ -4,15 +4,14 @@ Whole-body PBPK model of fluvoxamine as an inhibitor of CYP2C19 and CYP1A2
 This repository contains:
 
 - a PK-Sim snapshot (*.json) file of the current PBPK model
+- the PK-Sim project of the current PBPK model created with OSP version 10
```

Comment:
https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Model/pull/17#issuecomment-1076359418

Reply:
See comment at the end.